guide/getting-started/hardware.md

```diff
@@ -104,11 +106,11 @@ Bitcoin mining has turned into a professional undertaking years ago. Today, dedi
    height = 400
 %}
 
-Outside of purchasing hardware, the main cost of mining is electricity. That's why mining facilities can be found worldwide wherever electricity is abundant and cheap, whether in Iceland for geothermal energy or in China using excess hydroelectricity.
+Outside of purchasing hardware, the main cost of mining is electricity. That’s why mining facilities are found wherever electricity is abundant and cheap, such as in Iceland (geothermal) or in China (hydroelectricity).
```

Comment:
There is also a big benefit in that Bitcoin mining can turn localized / waste electricity into bitcoin (value).

Reply:
Linking to some additional material could be helpful here. I liked Nic Carters presentation at the B word conference, he did a very thorough analysis there. Unfortunately, the conference videos are not publicly available. Maybe we can link to the [Square/Ark paper](https://assets.ctfassets.net/2d5q1td6cyxq/2D2BnksJjavw4a6SUvAPwZ/c42a9e3a520b0cc3b230cda3b43eead5/BCEI_White_Paper_.pdf)?

Reply:
Need a concrete suggestion here where to put that link :)